common/source/docs/common-brushless-escs.rst

```diff
@@ -1,121 +1,94 @@
 .. _common-brushless-escs:
 
-====================================
-PWM,OneSHot, and DShot Protocol ESCs
-====================================
+================================
+PWM, OneShot and OneShot125 ESCs
+================================
 
-The most common electric motor used with ArduPilot vehicles is brushless and requires a brushless ESC for control. The main protocols for controlling these ESCs are PWM, OneShot, and DShot. In addition, some include the ability to be configured via an external PC application called BLHeli Suite or BLHeli32 Suite. See :ref:`common-blheli32-passthru` for more information on ESCs with that capability, but the following sections apply to ArduPilot's setup for those ESCs also.
+Most ArduPilot vehicles use brushless motors controlled by brushless motor ESCs. The most common protocols used by these ESCs are PWM, OneShot, OneShot125, :ref:`DShot <common-dshot-escs>` and :ref:`BLHeli <common-blheli32-passthru>`. This page describes the first three (PWM, OneShot and OneShot125).
+
+.. warning:: Be sure your ESC can support the configuration you select for it. Damage can occur otherwise. This includes frame rates discussed below. Also be careful when switching between protocols without rebooting or re-calibrating ESCs as this can lead to uncommanded motor output.
 
 PWM
 ===
 These are the most common ESCs for non-copter applications and were historically the first brushless ESCs.
 
 .. image:: ../../../images/hobbywing-esc.jpg
 
+PWM ESCs use a periodic input pulse of width typically between 1000uS and 2000uS for zero to full power, respectively.
```

Comment:
ok txs.  I also thought about this but shied away from changing it because it is existing wording and it is fairly complex.  In any case, I've added some wording for each vehicle now.